.azure-pipelines-templates/test.yml

```diff
@@ -7,6 +7,7 @@ steps:
     ./tests.sh -VV --timeout 240 --no-compress-output -T Test ${{ parameters.ctest_filter }}
   env:
     METRICS_MONGO_CONNECTION: $(METRICS_MONGO_CONNECTION)
+    TESTS_SUFFIX: ${{ parameters.suite_name_suffix }}
```

Comment:
Is this required?

Reply:
No it looks like we already have logic in add_perf_test but the results still don’t upload. I’m not convinced I prefer a larger CMake over using matrix.yml though.